docs/pages/_includes/services/security-service-keytab.md

```diff
@@ -0,0 +1,19 @@
+#### Place Service Keytab in DC/OS Secret Store
+
+The DC/OS {{ include.techName }} service uses a keytab containing all node principals (service keytab). After creating the principals above, generate the service keytab making sure to include all the node principals. This will be stored as a secret in the DC/OS Secret Store.
```

Comment:
It's difficult to see from this template, but does "creating the principals above" refer to valid principals?

Reply:
Yes